scripts/tests/chiptest/__init__.py

```diff
@@ -27,6 +30,179 @@
     os.path.join(_DEFAULT_CHIP_ROOT, "src/app/tests/suites"))
 
 
+@dataclass(eq=True, frozen=True)
+class ManualTest:
+    yaml: str
+    reason: str
+
+
+INVALID_TESTS = {
+    "tests.yaml",  # certification/tests.yaml is not a real test
+    "PICS.yaml",  # certification/PICS.yaml is not a real test
+}
+
+
+def _LoadManualTestsJson(json_file_path: str) -> Iterator[ManualTest]:
+    with open(json_file_path, 'rt') as f:
+        data = json.load(f)
+        for c in data["collection"]:
+            for name in data[c]:
+                yield ManualTest(yaml="%s.yaml" % name, reason=json_file_path)
+
+
+def _GetManualTests() -> Set[ManualTest]:
+    manualtests = set()
+
+    # Flaky tests? these seem to pass though
+    manualtests.add(ManualTest(yaml="DL_Schedules.yaml", reason="Flaky"))
+    manualtests.add(ManualTest(yaml="Test_TC_LVL_2_1.yaml", reason="Flaky"))
+
+    # Examples. Should these be manual?
+    manualtests.add(ManualTest(yaml="Config_Example.yaml", reason="Example"))
+    manualtests.add(ManualTest(yaml="Config_Variables_Example.yaml", reason="Example"))
+    manualtests.add(ManualTest(yaml="PICS_Example.yaml", reason="Example"))
+    manualtests.add(ManualTest(yaml="Response_Example.yaml", reason="Example"))
+    manualtests.add(ManualTest(yaml="Test_Example.yaml", reason="Example"))
+
+    # Flagged as manual from: src/app/tests/suites/manualTests.json
+    for item in _LoadManualTestsJson(os.path.join(_YAML_TEST_SUITE_PATH, "manualTests.json")):
```

Comment:
Suggest moving `_LoadManualTestsJson` this to the top with all other manualtests.add after with a comment saying that these are not actually manual tests, but just something we are using in the interim to stop these tests from running in CI until we work our the issues

Reply:
Moved and added comment.